filesystem/filesystem/config.py

```diff
@@ -0,0 +1,19 @@
+"""Filesystem shim app configuration."""
+
+import os
+import tempfile
+
+LEGACY_FILESYSTEM_ROOT = os.environ.get('LEGACY_FILESYSTEM_ROOT')
+if LEGACY_FILESYSTEM_ROOT is None:
+    LEGACY_FILESYSTEM_ROOT = tempfile.mkdtemp()
+
+LEGACY_FILESYSTEM_SOURCE_DIR_MODE = \
+    int(os.environ.get('LEGACY_FILESYSTEM_SOURCE_DIR_MODE', '17917'))
```

Comment:
What does mode `17917` translate to?

Reply:
In any case, it may be more legible as an octal string.

Reply:
``0o42775``. Fixed in 8ec0e04